aten/src/ATen/test/cuda_stream_test.cpp

```diff
@@ -145,14 +144,6 @@ TEST(TestStream, CUDAGuardTest) {
 
   // -- end setup
 
-  // Test that all original streams are recorded.
-  {
-    at::cuda::CUDAMultiStreamGuard guard;
```

Comment:
Are there any other tests covering `CUDAMultiStreamGuard`? If not, shall we add one?

Reply:
I grepped through the code and found only this one. I can see if I find a good place to add a few new ones.